HIP/hip-731.md

```diff
@@ -0,0 +1,59 @@
+---
+hip: 731
+title: Standarize the way to share scheduleId transactions
+author: Fran Fernandez <fran@kabila.app>, Adrian Marin <adrian@kabila.app>
+working-group: Fran Fernandez <fran@kabila.app>, Adrian Marin <adrian@kabila.app>, Rafael Lujan <rafa@kabila.app>
+type: Standards Track
+category: Application
+needs-council-approval: No
+status: Draft
+created: 2023-05-24
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/731
+---
+
+## Abstract
+
+This proposal aims to standardize the way of sharing the codes of a Schedule Transaction.
+
+## Motivation
+
+Right now if someone tries to share the id of a Schedule Transactions, for example from one dApp to different wallets, it is not supported as they are using different algorithms to obfuscate the generated code. We believe that this is a high barrier for users.
+
+## Rationale
+
+This is a new specification, so should not have alternative design.
+
+## User stories
+
+1. As an user that wants to sell nfts, I want to trade some nfts for hbar, so I want to generate a schedule transaction with nfts serial numbers and how much the receiver will pay. I want to get my code and share it with the buyer so he can execute it.
```

Comment:
When you say code are you talking about the memo?

Reply:
No, for code we mean an identifier of the scheduled transaction.